tests/test_orient_symmetric.py

```diff
@@ -27,60 +25,115 @@
     pytest.param(45, 3, np.float64, marks=pytest.mark.expensive),
 ]
 
+# For testing Cn methods where n>4.
+param_list_cn = [
+    (44, 5, np.float32),
+    pytest.param(44, 5, np.float32, marks=pytest.mark.expensive),
+    pytest.param(45, 6, np.float64, marks=pytest.mark.expensive),
+    pytest.param(44, 7, np.float32, marks=pytest.mark.expensive),
+    pytest.param(44, 8, np.float32, marks=pytest.mark.expensive),
+    pytest.param(45, 9, np.float64, marks=pytest.mark.expensive),
+]
+
 
 # Method to instantiate a Simulation source and orientation estimation object.
 def source_orientation_objs(L, n_img, order, dtype):
+    # This Volume is hand picked to have a fairly even distribution of energy.
```

Comment:
Since we are generating Cn volumes by rotating an initial set of Gaussians by the symmetry group we often end up with more of an accumulation near the center of the volume (especially if the initial set is close to the center). This seemed to be causing the algorithm to perform less optimally. For the tests I hand picked a volume where the gaussians were more evenly distributed in a spatial sense.

Garrett and I have discussed a few ways to address this during volume generation. I'll make an issue so we can circle back on it.

Reply:
Made issue #880

Reply:
Maybe "distribution of density" would be more accurate?

Reply:
Yes that makes sense.

Reply:
Done.